Add fluentbit integration

heitorPB opened the pull request:
**Please provide description of the purpose of this pull request, as well as its
motivation and context.**

Add fluentbit integration to slurmd

**How was the code tested?**

Use the following to configure fluentbit:

```bash
$ cat graylog-fluentbit
[{"output": [["name", "gelf"],
             ["match", "*"],
             ["host", "10.220.130.119"],
             ["port", "12201"],
             ["mode", "udp"],
             ["Gelf_Short_Message_Key", "log"]]}]

$ juju config fluentbit custom-config="$(cat graylog-fluentbit)"
```

Depends on https://github.com/omnivector-solutions/slurm-ops-manager/pull/68

Final checklist
- [x] I am the author of these changes, or I have the rights to submit them.
- [ ] I added the relevant changed to the README and/or documentation.
- [x] I self reviewed my own code.
- [x] I updated the `CHANGELOG` according to the [contributing
  guide](https://omnivector-solutions.github.io/osd-documentation/master/contributing.html#changelog)

dusktreader commented on charm-slurmd/lib/charms/fluentbit/v0/fluentbit.py:
```diff
@@ -0,0 +1,129 @@
+        """
+        super().__init__(charm, relation_name)
+
+        self._charm = charm
```
Is this necessary? I don't see where this reference is used.

heitorPB committed:
Add fluentbit integration

Get Fluentbit configuration parameters from slurm-ops-manager and send
it over the relation

Bump slurm-ops-manager from 0.6.13 to 0.7.0

heitorPB committed:
remove unnecessary files
